oauth2_provider/models.py

```diff
@@ -132,6 +132,10 @@ class AbstractApplication(models.Model):
     created = models.DateTimeField(auto_now_add=True)
     updated = models.DateTimeField(auto_now=True)
     algorithm = models.CharField(max_length=5, choices=ALGORITHM_TYPES, default=NO_ALGORITHM, blank=True)
+    allowed_origins = models.TextField(
+        blank=True,
+        help_text=_("Allowed origins list to enable CORS, space separated"),
```

Comment:
This help_text could be more informative.  Here is a suggestion. The details may not be accurate so it should be updated to reflect the actual requirements.

> Space-separated list of allowed origins for the token endpoint. The origin must be in the form of <scheme> "://" <host> [ ":" <port> ], such as https://login.mydomain.com or http://localhost:3000.  Query strings and hash information are not taken into account when validating these URLs. This does not include the 'Redirect URIs' or 'Post Logout Redirect URIs', if those domains will also use the token endpoint, they must be included in this list. 

We should decide whether the redirect uris are impicitly allowed origins or whether all allowed origins need to be explicitly specified. Also